packages/sanity/src/core/util/draftUtils.ts

```diff
@@ -153,13 +176,12 @@ export function collate<
   T extends {
     _id: string
     _type: string
-    _version?: Record<string, never>
   },
 >(documents: T[], {bundlePerspective}: CollateOptions = {}): CollatedHit<T>[] {
   const byId = documents.reduce((res, doc) => {
-    const isVersion = Boolean(doc._version)
-    const publishedId = getPublishedId(doc._id, isVersion)
-    const bundle = isVersion ? doc._id.split('.').at(0) : undefined
+    const publishedId = getPublishedId(doc._id)
+    const isVersion = isVersionId(doc._id)
+    const bundle = isVersion ? getBundleSlug(doc._id) : undefined
```

Comment:
COULD: `getBundleSlug` already does a `isVersionId` check, so could we just call it, and if following Bjorge's other suggestion ([here](https://github.com/sanity-io/sanity/pull/7357#discussion_r1718495759)) could simplify this

Reply:
Great point :)

Reply:
I'm keeping the isVersion since it's used in a condition lower down on this method and I'd prefer to keep it in a const vs just making the call directly in the if